src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/GeneralViewModel.cs

```diff
@@ -48,20 +54,24 @@ public GeneralViewModel(ISettingsRepository<GeneralSettings> settingsRepository,
 
             // set the callback function value to update the UI theme.
             UpdateUIThemeCallBack = updateTheme;
-            UpdateUIThemeCallBack(GeneralSettingsConfig.Theme.ToLower());
+
+            // Using InvariantCulture here as these are internal strings. However, FxCop
+            // also expects strings to be normalized to uppercase, so just for the
+            // purposes of this method we will operate on uppercase strings.
+            UpdateUIThemeCallBack(GeneralSettingsConfig.Theme.ToUpper(CultureInfo.InvariantCulture));
```

Comment:
CultureInfo.Ordinal.  I'll stop pointing this out everyone, but it would be great if you could do a pass on the rest of the changes.